src/vs/editor/contrib/find/simpleFindWidget.ts

```diff
@@ -93,13 +93,13 @@ export abstract class SimpleFindWidget extends Widget {
 
 		this._register(this._findInput.onKeyDown((e) => {
 			if (e.equals(KeyCode.Enter)) {
-				this.find(false);
+				this.find(true);
```

Comment:
I think this actually has to happen inside terminalFindWidget as simpleFindWidget is shared with the webview as well

Reply:
Also add an explanation comment where this switch is as it might be confusing if you're just reading the code